clang/runtime/dpct-rt/include/dpl_extras/algorithm.h.inc

```diff
@@ -2053,26 +2053,26 @@ inline ::std::pair<Iter1, Iter1> equal_range(_ExecutionPolicy &&policy,
 // DplExtrasVector|is_iterator
 // DPCT_DEPENDENCY_END
 // DPCT_CODE
-template <typename _ExecutionPolicy, typename key_t, typename key_out_t, 
+template <typename _ExecutionPolicy, typename key_t, typename key_out_t,
```

Comment:
I think we should keep the same style and use upper camel case type name, like `_KeyT`, can we rename this type name?

Reply:
https://github.com/oneapi-src/SYCLomatic/pull/902 should resolve styling / naming inconsistencies